ICRCs/ICRC-122/ICRC-122.md

```diff
@@ -0,0 +1,150 @@
+
+Each `122burn` or `122mint` block consists of the following top-level fields:
+
+| Field    | Type (ICRC-3 `Value`) | Required | Description |
+|----------|------------------------|----------|-------------|
+| `btype`  | `Text`                 | Yes      | MUST be one of: `"122burn"` or `"122mint"`. |
+| `ts`     | `Nat`                  | Yes      | Timestamp in nanoseconds when the block was added to the ledger. |
+| `phash`  | `Blob`                 | Yes      | Hash of the parent block. |
+| `tx`     | `Map(Text, Value)`     | Yes      | Encodes information about the token mint or burn transaction, including the caller. See schemas below. |
+
+### `tx` Field Schemas
+
+#### For `122burn`
+
+| Field        | Type (ICRC-3 `Value`)                                        | Required | Description |
+|--------------|--------------------------------------------------------------|----------|-------------|
+| `caller`     | `Value` (Must be `variant { Blob = <principal_bytes> }`)     | Yes      | The principal that invoked the ledger method (e.g., `icrc122_burn`) causing this block to be created. |
+| `from`       | `Value` (Must be `variant { Array = vec { V1 [, V2] } }`)¹ | Yes      | The account from which tokens are burned. |
+| `amount`     | `Nat`                                                        | Yes      | The number of tokens to burn. This value **MUST be greater than 0**. |
+| `reason`     | `Text`                                                       | Optional | Human-readable reason for the burn. |
+
+#### For `122mint`
+
+| Field        | Type (ICRC-3 `Value`)                                        | Required | Description |
+|--------------|--------------------------------------------------------------|----------|-------------|
+| `caller`     | `Value` (Must be `variant { Blob = <principal_bytes> }`)     | Yes      | The principal that invoked the ledger method (e.g., `icrc122_mint`) causing this block to be created. |
+| `to`         | `Value` (Must be `variant { Array = vec { V1 [, V2] } }`)¹ | Yes      | The account to which tokens are minted. |
+| `amount`     | `Nat`                                                        | Yes      | The number of tokens to mint. This value **MUST be greater than 0**. |
+| `reason`     | `Text`                                                       | Optional | Human-readable reason for the mint. |
+
+¹ Where `V1` is `variant { Blob = <owner_principal> }` and `V2` is `variant { Blob = <subaccount> }`. If no subaccount exists, the `Array` contains only `V1`.
```

Comment:
this seems to be duplicate information as already described in the "Common Elements" section. I think we can keep it, but I just want to make sure if it was a conscious decision to provide that information again here.

Reply:
Yes, it was intended like this. The first is for whoever reads the entire thing, the second is for whoever wants to immediately see the block types